target_score.txt

```diff
@@ -1,5 +1,9 @@
 # Target score for scrapper.py:
-6
+10
 
 # Target score for pipeline.py:
 0
+
+# Skip pipeline checks:
+1
```

Comment:
remove last four lines. ask @dmitry-uraev for reseting the file to upstream master while keeping your 10 as a desired mark